recipes/sdl/all/conanfile.py

```diff
@@ -449,3 +449,9 @@ def package_info(self):
 
             self.cpp_info.components["sdl2main"].libs = ["SDL2main" + postfix]
             self.cpp_info.components["sdl2main"].requires = ["libsdl2"]
+
+        # Workaround to avoid unwanted sdl::sdl target in CMakeDeps generator
+        self.cpp_info.set_property(
+            "cmake_target_name",
+            "SDL2::".format("SDL2main" if self.options.sdl2main else sdl2_cmake_target),
```

Comment:
```suggestion
            "SDL2::{}".format("SDL2main" if self.options.sdl2main else sdl2_cmake_target),
```
thanks to the linter